rfcs/rdm-0109-python-versions.md

```diff
@@ -0,0 +1,137 @@
+- Start Date: 2025-10-24
+- RFC PR: [#109](https://github.com/inveniosoftware/rfcs/pull/109)
+- Authors: Guillaume Viger
+
+# Python support in Invenio/RDM ecosystem
+
+## Summary
+
+This RFC outlines the Python support policy for InvenioRDM and the related Invenio ecocystem. In particular, it outlines how it relates to concrete implementations/use-cases. Some aspects are left flexible for agile decision making in future contexts.
+
+The general idea is:
+
+|                                 | Minimum                                       | Maximum                             |
+| ------------------------------- | --------------------------------------------- | ----------------------------------- |
+| Invenio framework level modules | Keep current minimum                          | Latest version officially supported |
```

Comment:
From my notes I am not sure if we meant keep those to the smallest Python version supported or to the current version they support. Some of them support 3.8 and other long deprecated versions...  My thoughts on that "Keep current minimum" are there: https://github.com/inveniosoftware/rfcs/pull/109/files#diff-ac3af67c596e1c277c361a8934a259cddd2c7be3fb9c1e6001a2f3ac5326d352R93-R100 .